src/main/java/pl/touk/ticketBooking/domain/Timetable/TimetableController.java

```diff
@@ -0,0 +1,64 @@
+package pl.touk.ticketBooking.domain.Timetable;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.touk.ticketBooking.domain.Bill.Bill;
+import pl.touk.ticketBooking.domain.Guest.Guest;
+import pl.touk.ticketBooking.domain.Movie.Movie;
+import pl.touk.ticketBooking.domain.Room.Room;
+import pl.touk.ticketBooking.domain.Room.RoomDto;
+import java.util.List;
+
+import static pl.touk.ticketBooking.domain.Timetable.TimetableDtoMapper.mapToTimeTableDtos;
+import static pl.touk.ticketBooking.domain.Room.RoomDtoMapper.mapToRoomDto;
+
+@RestController
+@RequiredArgsConstructor
+public class TimetableController {
+
+    private final TimetableService timetableService;
+
+    @GetMapping("/timetable")
+    public ResponseEntity<List<Timetable>> getTimetable(){
+        return timetableService.getTimetable();
+    }
+
+    @GetMapping(value = "/repertoire/{day}/{time}")
```

Comment:
W zadaniu jest punkt `The system lists movies available in the given time interval - title and screening
times.` - nie widzę tutaj tej funkcjonalności